scripts/copy-res.js

```diff
@@ -25,10 +25,13 @@ const INCLUDE_LANGS = [
     {'value': 'fi', 'label': 'Suomi'},
     {'value': 'fr', 'label': 'Français'},
     {'value': 'gl', 'label': 'Galego'},
+    {'value': 'hi', 'label': 'हिन्दी'},
     {'value': 'hu', 'label': 'Magyar'},
+    {'value': 'is', 'label': 'Íslensku'},
```

Comment:
What is your source for this spelling? I don't see it on https://en.wikipedia.org/wiki/Icelandic_language...